modules/ui/sections/background_list.js

```diff
@@ -61,6 +61,77 @@ export function uiSectionBackgroundList(context) {
       .append('ul')
       .attr('class', 'layer-list bg-extras-list');
 
+    const wayBackImageryEnter = bgExtrasListEnter
+      .append('li')
+      .attr('class', 'background.wayback_imagery.tooltip')
+      .append('div')
+      .style('display', 'flex');
+
+    const label = wayBackImageryEnter
+    .append('label')
+    .call(uiTooltip(context)
+      .title(l10n.t('background.wayback_imagery.tooltip'))
+      .placement('top')
+    );
+
+    label
+      .append('input')
+      .attr('type', 'checkbox')
+      .on('change', function(d3_event) {
+        d3_event.preventDefault();
+        dropdown.style('display', this.checked ? 'block' : 'none');
+
+        if (this.checked) {
+          imagery.getWaybackSource().then(sourceMap => {
```

Comment:
Have you verified that the information rendered on in the checkbox changes if you zoom far away? We want to make sure that `getWaybackSource()` returns different information during runtime as you pan around the earth. 

Otherwise, the info might be 'sticky' and never change.